Codigo/mdlCOmercioConUsuario.bas

```diff
@@ -102,14 +102,14 @@ Public Sub EnviarObjetoTransaccion(ByVal AQuien As Integer, ByVal UserIndex As I
                 Dim j As Long
                 'me fijo si tiene esas cantidades para que no duplique items
 108             For j = 1 To UBound(UserList(UserIndex).ComUsu.itemsAenviar)
-110                 If UserList(UserIndex).ComUsu.itemsAenviar(j).ObjIndex = ObjAEnviar.ObjIndex Then
+110                 If UserList(UserIndex).ComUsu.itemsAenviar(j).objIndex = ObjAEnviar.objIndex Then
 112                     cantidadTotalItem = cantidadTotalItem + UserList(UserIndex).ComUsu.itemsAenviar(j).amount
                     End If
 114             Next j
             
 116             cantidadTotalItem = cantidadTotalItem + ObjAEnviar.amount
             
-118             If Not TieneObjetos(ObjAEnviar.ObjIndex, cantidadTotalItem, UserIndex) Then
+118             If Not TieneObjetos(ObjAEnviar.objIndex, cantidadTotalItem, UserIndex) Then
```

Comment:
please rename this function from `TieneObjetos` to `has_objects` to avoid the problem with VB6 changing uppercase to lowercase automatically like above.